fuzz/fuzz_targets/spectests.rs

```diff
@@ -0,0 +1,33 @@
+#![no_main]
+
+use arbitrary::Arbitrary;
+use libfuzzer_sys::fuzz_target;
+use wasmtime::{Config, Engine, Store};
+use wasmtime_fuzzing::generators::OptLevel;
+use wasmtime_wast::WastContext;
+
+include!(concat!(env!("OUT_DIR"), "/spectests.rs"));
+
+#[derive(Arbitrary, Debug)]
+pub struct SpecCompliantConfig {
```

Comment:
This should be in `wasmtime/crates/fuzzing/src/generators.rs` with the rest of the test case generators. This allows us to reuse the concept "a spec compliant config" across different oracles, and is consistent with the rest of the infra.